src/ontology/eupath_dev.owl

```diff
@@ -6785,6 +6785,19 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0000815 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0000815">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0200000"/>
+        <obo:IAO_0000115>A data transformation that uses data from a whole metagenome sequencing assay and CORRAL (for Clustering of Reference Alignments) software to specify taxa of Eukaryota in a specimen.</obo:IAO_0000115>
```

Comment:
```suggestion
        <obo:IAO_0000115>A data transformation that uses data from a whole metagenome sequencing assay and CORRAL (for Clustering Of Related Reference ALignments) software to specify taxa of Eukaryota in a specimen.</obo:IAO_0000115>
```

Reply:
(for reference, the above change was just copied from the [paper](https://www.biorxiv.org/content/10.1101/2022.03.09.483664v2))